nmdc_runtime/site/export/ncbi_xml.py

```diff
@@ -203,6 +218,10 @@ def set_biosample(
 
         # Process individual biosamples
         for biosample in individual_biosamples:
+            # Skip if biosample has INSDC identifiers
+            if self._has_insdc_biosample_identifier(biosample):
+                continue
```

Comment:
If individual biosample has `insdc_biosample_identifiers` asserted on it and is not empty, do not create BioSample `<Action>` block for it.